extensions/interactions/base-interaction-validation.service.ts

```diff
@@ -28,7 +28,7 @@ import { AppConstants } from 'app.constants';
 
 export interface Warning {
   type: string;
-  message: string
+  message: string;
```

Comment:
Unneeded change.

Reply:
Will Remove that Sir. Sir  Could you please tell me how can i perform tests locally. I tried npm test and also npm run tsc(After installing tsc).